src/slam_mapper.cpp

```diff
@@ -207,6 +207,12 @@ void SMapper::configure(const rclcpp::Node::SharedPtr & node)
       correlation_search_space_dimension);
   }
   node->get_parameter("correlation_search_space_dimension", correlation_search_space_dimension);
+  if (correlation_search_space_dimension <= 0) {
```

Comment:
Did you catch all of the error conditions in settings?

Reply:
I catch all error conditions that can cause #433